packages/integration-tests/projects/suite-web/tests/coinmarket/exchange.test.ts

```diff
@@ -0,0 +1,206 @@
+// @group:coinmarket
+
+describe('Coinmarket exchange', () => {
+    beforeEach(() => {
+        cy.task('startEmu', { wipe: true });
+        cy.task('setupEmu', {
+            needs_backup: false,
+            mnemonic:
+                'alcohol woman abuse must during monitor noble actual mixed trade anger aisle',
+        });
+        cy.task('startBridge');
+
+        cy.viewport(1080, 1440).resetDb();
+        cy.interceptInvityApi();
+        cy.prefixedVisit('/');
+        cy.passThroughInitialRun();
+        cy.discoveryShouldFinish();
+        cy.enableRegtestAndGetCoins({
+            payments: [
+                {
+                    address: 'bcrt1qnspxpr2xj9s2jt6qlhuvdnxw6q55jvyg6q7g5r',
+                    amount: 1,
+                },
+            ],
+        });
+
+        // Enables ETH account
+        cy.getTestElement('@suite/menu/settings').click();
+        cy.getTestElement('@settings/menu/wallet').click();
+        cy.getTestElement('@settings/wallet/network/eth').click();
+        cy.getTestElement('@settings/menu/close').click();
+
+        // Goes to Exchange
+        cy.getTestElement('@suite/menu/wallet-index').click();
+        cy.getTestElement('@account-menu/regtest/normal/0/label').click();
+        cy.getTestElement('@wallet/menu/wallet-coinmarket-buy').click();
+        cy.getTestElement('@coinmarket/menu/wallet-coinmarket-exchange').click();
+    });
+
+    /**
+     * Test case
+     * 1. Go to Accounts/REGTEST account/Trade/Exchange
+     * 2. Check whether all input windows are empty and crypto input is set on REGTEST
+     * 3. Exchange 0.005REGTEST for ETH with custom fee of 1 sat
+     * 4. Verifies the mocked offers are all fully and correctly displayed
+     * 5. Pick one offer
+     * 6. Verifies the amounts, currencies, addresses and providers are all in accordance with the mock
+     * 7. Confirms the transaction and verifies the same information in the modal
+     * 8. Return back to the Exchange tab in Trezor Suite
+     */
+
```

Comment:
Remove the blank line; the test case documentation should undoubtly belong to the `it`.